.github/workflows/test.yml

```
@@ -0,0 +1,81 @@
+
+      - name: Get Dependencies
+        run: flutter pub get
+
+      - name: Generate Localization Files
+        run: flutter gen-l10n
+
+      - name: Build Runner (Generate Dart Files)
+        run: dart run build_runner build --delete-conflicting-outputs
+
+      - name: Format Check (Exclude generated files)
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:2:3: Quote this to prevent word splitting

(shellcheck)